event-sourcing/subscription-manager/src/main/java/uk/gov/justice/services/event/sourcing/subscription/SubscriptionManagerStartUp.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.justice.services.event.sourcing.subscription;
+
+import uk.gov.justice.services.subscription.SubscriptionManager;
+import uk.gov.justice.subscription.domain.subscriptiondescriptor.Subscription;
+import uk.gov.justice.subscription.domain.subscriptiondescriptor.SubscriptionDescriptorDefinition;
+import uk.gov.justice.subscription.registry.SubscriptionDescriptorDefinitionRegistry;
+
+import java.util.List;
+import java.util.Set;
+
+import javax.annotation.PostConstruct;
+import javax.ejb.Singleton;
+import javax.ejb.Startup;
+import javax.enterprise.inject.Any;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Startup
+@Singleton
+public class SubscriptionManagerStartUp {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SubscriptionManagerStartUp.class);
+
+    @Inject
+    @Any
+    private Instance<SubscriptionManager> subscriptionManagers;
```

Comment:
Injection points on the framework are usually not private, but package protected

Reply:
updated